integration-tests/tests/list_test.go

```diff
@@ -42,6 +42,7 @@ type listSuite struct {
 
 var verRegexp = `(\d{2}\.\d{2}.*|\w{12})`
 
+// ported to spread: tests/listing
 func (s *listSuite) TestListMustPrintCoreVersion(c *check.C) {
```

Comment:
I think we should actually move over instead of just adding a note. In other words, when we put the test on spread, we remove it from integration-tests/...